packages/react/src/primitives/SearchField/SearchFieldSuggestionMenu.tsx

```diff
@@ -0,0 +1,224 @@
+      <ScrollView
+        as="ul"
+        role="listbox"
+        ariaLabel={ariaLabel}
+        className={ComponentClassNames.SearchFieldMenuSuggestions}
+        id={id}
+      >
+        {suggestions.map((suggestion: Suggestion, idx) => {
+          const { id, label } = suggestion;
+          const isActive = activeIdx === idx;
+
+          const handleOnClick: React.MouseEventHandler<HTMLLIElement> = () => {
+            setIsMenuOpen(false);
+            setActiveIdx(-1);
+            if (!isControlled) {
+              setInternalValue(label);
+            }
+            if (isFunction(onSuggestionSelect)) {
+              onSuggestionSelect(suggestion);
+            }
+          };
+
+          const handleOnMouseDown: React.MouseEventHandler<HTMLLIElement> = (
+            event
+          ) => {
+            event.preventDefault();
+          };
+
+          const handleOnMouseMove: React.MouseEventHandler<HTMLLIElement> =
+            () => {
+              setActiveIdx(idx);
+            };
+
+          const suggestionId = `${suggestionBaseId}-suggestion-${idx}`;
+
+          return (
+            <SearchFieldSuggestion
+              aria-selected={isActive}
+              className={classNameModifierByFlag(
+                ComponentClassNames.SearchFieldMenuSuggestion,
+                'active',
+                isActive
+              )}
+              id={id || suggestionId}
+              key={id || suggestionId}
+              onClick={handleOnClick}
+              onMouseDown={handleOnMouseDown}
+              onMouseMove={handleOnMouseMove}
+            >
+              {isFunction(renderSuggestion) ? (
+                renderSuggestion(suggestion, value)
+              ) : filteringType !== 'manual' ? (
+                <HighlightMatch query={value}>{label}</HighlightMatch>
+              ) : (
+                label
+              )}
+            </SearchFieldSuggestion>
+          );
+        })}
+      </ScrollView>
+    ) : (
+      <NoSuggestions />
+    );
+  }, [
+    NoSuggestions,
+    activeIdx,
+    ariaLabel,
+    filteringType,
+    id,
+    isControlled,
+    onSuggestionSelect,
+    renderSuggestion,
+    setActiveIdx,
+    setInternalValue,
+    setIsMenuOpen,
+    suggestionBaseId,
+    suggestions,
+    value,
+  ]);
+
+  React.useEffect(() => {
+    const menuContainerElement = document.getElementById(menuContainerId);
+    if (menuContainerElement && isOpen) {
+      const { top, bottom } = menuContainerElement.getBoundingClientRect();
+
+      if (top < 0 || bottom > document.documentElement.clientHeight) {
+        window.scrollTo({
+          top:
+            bottom -
+            document.documentElement.clientHeight +
+            window.scrollY +
+            20,
+          behavior: 'smooth',
+        });
+      }
+    }
+  }, [isOpen, menuContainerId]);
+
+  React.useEffect(() => {
+    const activeSuggestionElement = document.getElementById(activeSuggestionId);
```

Comment:
This will make it able to scroll with keyboard
![autocomplete_keyboard](https://user-images.githubusercontent.com/40295569/194901261-773c319f-14d4-4ea6-9bbf-974d0cec04db.gif)